modules/claims_api/spec/lib/claims_api/poa_pdf_constructor/individual_spec.rb

```diff
@@ -45,7 +45,7 @@
     Timecop.return
   end
 
-  xit 'construct pdf' do
+  it 'construct pdf' do
```

Comment:
We shouldn't re-enable this spec as part of this PR.
Somebody disabled it in the past because it was flaky.
If we want to re-enable it, we should create a ticket to go investigate if it's ok to re-enable and make that change (and only that change).